conf.d/health.d/ipfs.conf

```diff
@@ -0,0 +1,11 @@
+
+   alarm: ipfs_datastore_in_use
+      on: ipfs_local.repo_size
```

Comment:
To be clear (I'm a bit new to netdata alarms), I need to change
`on: ipfs_local.repo_size` to be `template: ipfs_local.repo_size` ?